cra-config.yaml

```diff
@@ -0,0 +1,6 @@
+# More info about this file at https://github.com/terraform-ibm-modules/common-pipeline-assets/blob/main/.github/workflows/terraform-test-pipeline.md#cra-config-yaml
+version: "v1"
+CRA_TARGETS:
+  - CRA_TARGET: "examples/default" # Target directory for CRA scan. If not provided, the CRA Scan will not be run.
+    CRA_IGNORE_RULES_FILE: "cra-tf-validate-ignore-rules.json" # CRA Ignore file to use. If not provided, it checks the repo root directory for `cra-tf-validate-ignore-rules.json`
+    PROFILE_ID: "262b5a6d-9dea-400e-b61f-0fbd63883f78" # SCC profile ID (currently set to the FSCloud 1.3.0 profile).
```

Comment:
is this correct? I thought we where on 1.4?

Reply:
I dont want to move to 1.4.0 yet because it wont get picked up in this PR (since it pulls config from master branch). Ill update it in next PR when the config in this yaml gets read